app/helpers/organisation_helper.rb

```diff
@@ -40,11 +35,11 @@ def ofsted_report(school)
   end
 
   def age_range(school)
-    if school.minimum_age && school.maximum_age?
-      "#{school.minimum_age} to #{school.maximum_age}"
-    else
-      I18n.t('schools.not_given')
-    end
+    age_range_or_nil(school) || I18n.t('schools.not_given')
```

Comment:
I would rewrite this method like this:

```
def age_range(school)
  return I18n.t('schools.not_given') unless school.minimum_age? && school.maximum_age?
  "#{school.minimum_age} to #{school.maximum_age}"
end
```

Reply:
I was trying to avoid duplicating `school.minimum_age? && school.maximum_age?`, but maybe it's OK to duplicate something so small.

Reply:
I found a way to do this without using a method age_range_or_nil - I reject `I18n.t('schools.not_given')` in `organisation_type`